Runtime/Scripts/LocationService.cs

```diff
@@ -110,29 +100,27 @@ public static Loc RetrieveLocation()
     {
       if (Input.location.status != LocationServiceStatus.Running)
       {
-        UnityEngine.Debug.Log("Warning: Location Status must be in running state to get a valid location! Current Status: " + Input.location.status);
+        Debug.LogWarning("MobiledgeX: Location Status must be in running state to get a valid location! Current Status: " + Input.location.status);
       }
       LocationInfo locationInfo = Input.location.lastData;
-      UnityEngine.Debug.Log("Location Info: [" + locationInfo.longitude + "," + locationInfo.latitude + "]");
+      Debug.Log("Location Info: [" + locationInfo.longitude + "," + locationInfo.latitude + "]");
```

Comment:
Maybe we don't have to print. The location use is just to trigger the Permissions dialog. Might still be useful though.

Reply:
I like having it for debug purposes, but not a hard requirement